packages/project/package.json

```diff
@@ -28,7 +30,14 @@
     "tsup": "^6.1.3",
     "typescript": "4.7.4"
   },
-  "main": "lib/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "require": "./lib/cjs/index.cjs",
+      "default": "./lib/index.js"
```

Comment:
With those set of conditions `default` seems to be redundant. I don't exactly remember the very first rules around conditions in node before they were adjusted but I don't think that you have to worry about some old node@12 versions here.